prow/scripts/governance.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly ARGS=("$@")
+readonly MILV_IMAGE="magicmatatjahu/milv:0.0.6"
+VOLUME_DIR=""
+OUTPUT=0
+
+function read_arguments() {
+    # read arguments
+    while test $# -gt 0; do
+        case "$1" in
+            --repository)
+                shift
+                readonly REPOSITORY_NAME=$1
+                shift
+                ;;
+            --repository-org)
+                shift
+                readonly REPOSITORY_ORG=$1
+                shift
+                ;;
+            --repository-dir)
+                shift
+                readonly REPOSITORY_DIR=$1
+                shift
+                ;;
+            --full-validation)
+                shift
+                readonly FULL_VALIDATION=$1
+                shift
+                ;;
+            *)
+                echo "$1 is not a recognized flag!"
+                exit 1;
+                ;;
+        esac
+    done
+
+    if [[ -z "${REPOSITORY_NAME}" ]]; then
+        echo -e "ERROR: repository name is required"
+        exit 1
+    fi
+
+    if [[ -z "${REPOSITORY_ORG}" ]]; then
+        REPOSITORY_ORG="kyma-project"
+    fi
+
+    if [[ -z "${REPOSITORY_DIR}" ]]; then
+        REPOSITORY_DIR="${PWD}"
+    fi
+
+    VOLUME_DIR="${REPOSITORY_DIR}"
+}
+
+function load_library() {
+    local script_dir=
+    script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    # shellcheck disable=SC1090
+    source "${script_dir}/library.sh"
+}
+
+function fetch_origin_master() {
+    local repository=""
```

Comment:
You can define it in one line., also in other places.
Defining it separately is required when you assign execution output to a variable.